packages/esm-patient-tests-app/src/test-results/tree-view/tree-view-wrappper.component.test.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import TreeViewWrapper from './tree-view-wrapper.component';
+import { useGetManyObstreeData } from '../grouped-timeline';
+import { useConfig } from '@openmrs/esm-framework';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  useConfig: jest.fn(),
+  useLayoutType: jest.fn(),
+  createGlobalStore: jest.fn(),
+  createUseStore: jest.fn(),
+}));
+
+// Mock @openmrs/esm-patient-common-lib
+jest.mock('@openmrs/esm-patient-common-lib', () => ({
+  EmptyState: jest.fn(({ headerTitle, displayText }) => (
+    <div data-testid="mock-empty-state">
+      <div>{headerTitle}</div>
+      <div>{displayText}</div>
+    </div>
+  )),
+  ErrorState: jest.fn(({ headerTitle }) => (
+    <div data-testid="mock-error-state">
+      <div>{headerTitle}</div>
+    </div>
+  )),
+}));
+
+jest.mock('../grouped-timeline', () => ({
+  useGetManyObstreeData: jest.fn(),
+}));
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key,
+  }),
+}));
+
+jest.mock('./tree-view.component', () => {
+  return jest.fn().mockImplementation((props) => {
+    return (
+      <div data-testid="mock-tree-view">
+        TreeView Component
+        <div data-testid="tree-view-props">{JSON.stringify(props)}</div>
+      </div>
+    );
+  });
+});
+
+const mockConfig = {
+  resultsViewerConcepts: [{ conceptUuid: 'concept-1' }, { conceptUuid: 'concept-2' }],
+};
```

Comment:
https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#annotating-useconfig-mocks-with-type-information